docs/how-to/enable-real-time-ubuntu.rst

```diff
@@ -1,141 +1,4 @@
-   Refer to :doc:`../reference/releases` for details.
-
-Choose Generic or a corresponding variant:
-
-.. tabs::
-
-   .. group-tab:: Generic
-
-      .. code-block:: shell
-
-         sudo pro enable realtime-kernel
-      
-      .. caution::
-         The generic realtime kernel is not intended for Raspberry Pi.
-         Using the Pro client to install it on these platforms will make the system unusable.
-
-   .. group-tab:: Raspberry Pi
-      For Raspberry Pi 4 and 5:
-
-      .. code-block:: shell
-
-         sudo pro enable realtime-kernel --variant=raspi
-
-   .. group-tab:: Intel IOTG
-      For 12th Gen Intel® Core™ processors:
-
-      .. code-block:: shell
-
-         sudo pro enable realtime-kernel --variant=intel-iotg
-
-You'll need to acknowledge a warning, then you should see confirmation that the
-Real-time Ubuntu package is installed:
-
-.. code-block:: text
-
-   One moment, checking your subscription first
-   Real-time kernel cannot be enabled with Livepatch.
-   Disable Livepatch and proceed to enable Real-time kernel? (y/N) y
-   Disabling incompatible service: Livepatch
-   The Real-time kernel is an Ubuntu kernel with PREEMPT_RT patches integrated.
-
-   This will change your kernel. To revert to your original kernel, you will need
-   to make the change manually.
-
-   Do you want to continue? [ default = Yes ]: (Y/n) Y
-   Updating Real-time kernel package lists
-   Updating standard Ubuntu package lists
-   Installing Real-time kernel packages
-   Real-time kernel enabled
-   A reboot is required to complete install.
-
-After rebooting, you'll be running Real-time Ubuntu.
-
-Install and enable manually
----------------------------
-
-To access the Real-time Ubuntu repository but not install the package
-immediately, first use the ``--access-only`` flag:
-
-.. code-block:: shell-session
-
-   $ sudo pro enable realtime-kernel --access-only
-
-   One moment, checking your subscription first
-   Real-time kernel cannot be enabled with Livepatch.
-   Disable Livepatch and proceed to enable Real-time kernel? (y/N) y
-   Disabling incompatible service: Livepatch
-   Updating Real-time kernel package lists
-   Skipping installing packages: ubuntu-realtime
-   Real-time kernel access enabled
-
-.. important::
-
-   The ``--access-only`` flag was introduced in Pro Client version 27.11.
-
-Now that Real-time Ubuntu is accessible, you can install and enable it whenever
-you wish.
-
-For example, to install the generic realtime kernel (not suitable for Raspberry Pi):
-
-.. code-block:: shell
-
-   sudo apt install ubuntu-realtime
-
-
-After rebooting, you'll be running Real-time Ubuntu.
-
-
-.. LINKS
-.. _Real-time Ubuntu: https://ubuntu.com/real-time
-.. _Ubuntu Pro: https://ubuntu.com/pro
-.. _compatible with Real-time Ubuntu: https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/references/compatibility_matrix/
+.. include:: /reuse/install-rtu.txt
```

Comment:
decided to use an include file for easier maintenance.
the include file is txt (not rst/md) so that Sphinx doesn't return warnings that the file is not in any TOC tree.